sdks/python/apache_beam/runners/common.pxd

```diff
@@ -86,15 +86,16 @@ cdef class DoFnRunner(Receiver):
 
 
 cdef class OutputProcessor(object):
-  @cython.locals(windowed_value=WindowedValue)
+  @cython.locals(windowed_value=WindowedValue, output_element_output=cython.int)
```

Comment:
<!--new_thread; commit:6c8d60bbfb2732c20d13ce00cddb83e7be73cd40; resolved:0-->
You can do `cdef bint has_per_element_output_count` here.

Reply:
I'm not sure whether I understand this comment correctly, but I defined 'has_per_element_output_count' in 'cdef class _OutputProcessor', which is a derived class of OutputProcessor

Reply:
Ah, my mistake.